openff/toolkit/topology/molecule.py

```diff
@@ -351,7 +372,12 @@ def element(self):
         -------
         openmm.openmm.app.element.Element
         """
-        return element.Element.getByAtomicNumber(self._atomic_number)
+        try:
+            from openmm.app.element import Element
+        except ImportError:
+            from simtk.openmm.app.element import Element
+
+        return Element.getByAtomicNumber(self._atomic_number)
```

Comment:
Hmm, this seems like a big loose end. At the start of the file we import `mdtraj`'s `Element` package, so this, in effect, actually makes us hard-dependent on both openMM and MDTraj. Do we have a plan for which periodic table we'll ultimately use? Pint doesn't have one, QCElemental seems to package its own, and I've heard talk about the lightweight mendeleev package as well. I suppose my first inclination is to stick with mdtraj but it seems kinda heavyweight for the job.

Reply:
No strong opinions here, my go-to lightweight elements package is [`ele`](https://github.com/rsdefever/ele), though `mendeleev` seems plenty lightweight itself:

> depends | colorama, numpy, pandas, pip, pyfiglet, pygments, python >=3.5, sqlalchemy

I guess I'm fine with either option: stick with MDTraj here or migrate now/soon.

Reply:
Here's what it pulls down (from my development environment):

```shell
$ mamba install mendeleev -c conda-forge                         openmm-optional  ✱

                  __    __    __    __
                 /  \  /  \  /  \  /  \
                /    \/    \/    \/    \
███████████████/  /██/  /██/  /██/  /████████████████████████
              /  / \   / \   / \   / \  \____
             /  /   \_/   \_/   \_/   \    o \__,
            / _/                       \_____/  `
            |/
        ███╗   ███╗ █████╗ ███╗   ███╗██████╗  █████╗
        ████╗ ████║██╔══██╗████╗ ████║██╔══██╗██╔══██╗
        ██╔████╔██║███████║██╔████╔██║██████╔╝███████║
        ██║╚██╔╝██║██╔══██║██║╚██╔╝██║██╔══██╗██╔══██║
        ██║ ╚═╝ ██║██║  ██║██║ ╚═╝ ██║██████╔╝██║  ██║
        ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝     ╚═╝╚═════╝ ╚═╝  ╚═╝

        mamba (0.18.2) supported by @QuantStack

        GitHub:  https://github.com/mamba-org/mamba
        Twitter: https://twitter.com/QuantStack

█████████████████████████████████████████████████████████████


Looking for: ['mendeleev']

pkgs/r/osx-64            [====================] (00m:00s) No change
pkgs/r/noarch            [====================] (00m:00s) No change
pkgs/main/noarch         [====================] (00m:00s) Done
pkgs/main/osx-64         [====================] (00m:02s) Done
conda-forge/noarch       [====================] (00m:03s) Done
conda-forge/osx-64       [====================] (00m:08s) Done

Pinned packages:
  - python 3.9.*


Transaction

  Prefix: /Users/mwt/miniconda3/envs/openff-dev

  Updating specs:

   - mendeleev
   - ca-certificates
   - certifi
   - openssl


  Package        Version  Build         Channel                  Size
───────────────────────────────────────────────────────────────────────
  Install:
───────────────────────────────────────────────────────────────────────

  + mendeleev      0.9.0  pyh8a188c0_0  conda-forge/noarch     156 KB
  + pyfiglet   0.8.post1  py_0          conda-forge/noarch     514 KB

  Summary:

  Install: 2 packages

  Total download: 671 KB

───────────────────────────────────────────────────────────────────────

Confirm changes: [Y/n]

Reply:
Okay - switched over to `mendeleev` in b54d78ef409c62d131cb8c1607bce12c0153299f - this also drops MDTraj as a hard dependency!

Reply:
Also FTR: I only learned today that MDTraj vendors OpenMM for its element module. So that's two moving pieces that can be collapsed down to one.

Reply:
Hah, I didn't know that about MDTraj. Ok, mendeleev looks like a good call here (more activity+stars than ele). Thanks!

Reply:
FYI, we are already using mendeleev for creating the substructures from CIF files in the topology refactor.

Reply:
Ahhhhh, great catch - YOU'RE where I heard about it from. Thanks for doing the legwork on that search, @ijpulidos - It looks like a great choice.